pandas/io/data.py

```diff
@@ -1,1247 +1,6 @@
-        """
-
-        url = self._OPTIONS_BASE_URL.format(sym=self.symbol)
-        root = self._parse_url(url)
-
-        try:
-            links = root.xpath('//*[@id="options_menu"]/form/select/option')
-        except IndexError:
-            raise RemoteDataError('Expiry dates not available')
-
-        expiry_dates = [dt.datetime.strptime(element.text, "%B %d, %Y").date() for element in links]
-        links = [element.attrib['data-selectbox-link'] for element in links]
-
-        if len(expiry_dates) == 0:
-            raise RemoteDataError('Data not available')
-
-        expiry_links = dict(zip(expiry_dates, links))
-        self._expiry_links = expiry_links
-        self._expiry_dates = expiry_dates
-        return expiry_dates, expiry_links
-
-    def _parse_url(self, url):
-        """
-        Downloads and parses a URL, returns xml root.
-
-        """
-        try:
-            from lxml.html import parse
-        except ImportError:
-            raise ImportError("Please install lxml if you want to use the "
-                              "{0!r} class".format(self.__class__.__name__))
-        try:
-            doc = parse(url)
-        except _network_error_classes:
-            raise RemoteDataError("Unable to parse URL "
-                                  "{0!r}".format(url))
-        else:
-            root = doc.getroot()
-            if root is None:
-                raise RemoteDataError("Parsed URL {0!r} has no root"
-                                      "element".format(url))
-        return root
-
-    def _process_data(self, frame, type):
-        """
-        Adds columns for Expiry, IsNonstandard (ie: deliverable is not 100 shares)
-        and Tag (the tag indicating what is actually deliverable, None if standard).
-
-        """
-        frame.columns = ['Strike', 'Symbol', 'Last', 'Bid', 'Ask', 'Chg', 'PctChg', 'Vol', 'Open_Int', 'IV']
-        frame["Rootexp"] = frame.Symbol.str[0:-9]
-        frame["Root"] = frame.Rootexp.str[0:-6]
-        frame["Expiry"] = to_datetime(frame.Rootexp.str[-6:])
-        #Removes dashes in equity ticker to map to option ticker.
-        #Ex: BRK-B to BRKB140517C00100000
-        frame["IsNonstandard"] = frame['Root'] != self.symbol.replace('-', '')
-        del frame["Rootexp"]
-        frame["Underlying"] = self.symbol
-        try:
-            frame['Underlying_Price'] = self.underlying_price
-            frame["Quote_Time"] = self.quote_time
-        except AttributeError:
-            frame['Underlying_Price'] = np.nan
-            frame["Quote_Time"] = np.nan
-        frame.rename(columns={'Open Int': 'Open_Int'}, inplace=True)
-        frame['Type'] = type
-        frame.set_index(['Strike', 'Expiry', 'Type', 'Symbol'], inplace=True)
-
-        return frame
+raise ImportError(
+    "The pandas.io.data module is moved to a separate package "
+    "(pandas-datareader). After installing the pandas-datareader package "
```

Comment:
yep, added an entry in the overview at https://github.com/pydata/pandas/issues/6581